src/math/Box3.js

```diff
@@ -1,59 +1,24 @@
 import { Vector3 } from './Vector3.js';
 
-const _points = [
-	new Vector3(),
-	new Vector3(),
-	new Vector3(),
-	new Vector3(),
-	new Vector3(),
-	new Vector3(),
-	new Vector3(),
-	new Vector3()
-];
-
-const _vector = new Vector3();
-
-const _box = new Box3();
```

Comment:
internal values have been moved to the end of the file, since this calling Box3 itself.